### PR TITLE
Docs revision

### PR DESCRIPTION
**Pull request by VictorAlbertos**
I will hold this PR until Wiebke tell me what email we should add for comercial information.

### CODE REVIEW

**Comment by chemarubio** [comment]
I asked Caroline about this last week, and she was going to share with sales people. Once that was done, she will get back to me so I asked for the recco@significo.com email to be generated.

File: README.md [hunk 1]
```diff
@@ -239,8 +246,8 @@ Button( // Button config here ),
 [Github-Packages]:https://github.com/features/packages
 [PAT]:https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens
 [Github-Recco]:https://github.com/orgs/viluahealthcare/packages?repo_name=recco-android-sdk
```
Comment (miguelbcr) [inline README.md:248]: should this point out to https://github.com/orgs/sf-recco/packages?repo_name=android-sdk instead?

and probably https://github.com/viluahealthcare/recco-ios-sdk too eventually

**Review by miguelbcr — APPROVED** [review]
thanks! 🚀

File: README.md (hunk 1 above)
Comment (chemarubio) [reply README.md:248]: iOS link is now broken, but will be working as soon as our colleagues migrate.

**Pull request merged**